javascript/ql/src/semmle/javascript/Expr.qll

```diff
@@ -267,13 +321,44 @@ class ParExpr extends @parexpr, Expr {
   override Expr getUnderlyingReference() { result = getExpression().getUnderlyingReference() }
 }
 
-/** A `null` literal. */
+/**
+ * A `null` literal.
+ *
+ * Example:
+ *
+ * ```
+ * null
+ * ```
+ */
 class NullLiteral extends @nullliteral, Literal { }
 
-/** A Boolean literal, that is, either `true` or `false`. */
+/**
+ * A Boolean literal, that is, either `true` or `false`.
+ *
+ * Examples:
+ *
+ * ```
+ * true
+ * false
+ * ```
+ */
 class BooleanLiteral extends @booleanliteral, Literal { }
 
-/** A numeric literal. */
+/**
+ * A numeric literal.
+ *
+ * Examples:
+ *
+ * ```
+ * 0b101
+ * 0o31
+ * 25
+ * 0xffff
+ * 6.626e-34
+ * Infinity
+ * NaN
```

Comment:
Are `Infinity` and `NaN` actually extracted as number literals? I think they're just seen as global variables.

Reply:
Er, yes, you are right.